contrib/fuzz/containerd_import_fuzzer.go

```diff
@@ -40,14 +40,19 @@ var (
 	initDaemon sync.Once
 )
 
+func startDaemonForFuzzing(arguments []string) {
+	app := command.App()
```

Comment:
Note to reviewers: This fuzzer is starting containerd by instantiating command.App directly. To have all containerd's plugins, it has to have all of `_ "github.com/containerd/containerd/.../plugin"` import and that's why we need `builtins` package.

Reply:
@AdamKorcz Regarding directly instantiating command.App, does it help coverage-guided fuzzers to know more about what is covered?

Reply:
> Regarding directly instantiating fuzzers, does it help coverage-guided fuzzers to know more about containerd, compared to launching containerd as an external process?

Yes. Interacting with a binary executable is less effective than the approach of launcing `command.App`